### PR TITLE
Add cREAL token and refactor for easier token additions

### PR DESCRIPTION
**Pull request by aaronmgdr**
* add instructions `adding-stabecoins.md`
* update contract kit
* add cREAL svg
* fetch all currencies, convert to be their value against USD instead of against EUR
* have one function for fetching. any celo stable rather than a function per token
* refactor such that adding a token requires just adding to the config rather than creating 5 new functions.

### CODE REVIEW

File: src/providers/ExchangeRateAPI.test.ts [hunk 1]
```diff
@@ -1,13 +1,188 @@
...
+    const price = await usdToFiat()
+    expect(price).toEqual(
+      expect.objectContaining({
+        source: "exchangeratesapi.io",
+        time: 1614211200000,
+        hasError: false,
+      })
+    )
+    expect(price).toEqual(
+      expect.objectContaining({
```
Comment (Henrynunez112) [inline src/providers/ExchangeRateAPI.test.ts:14]: Are these values hard coded??

**Review by Henrynunez112 — APPROVED** [review]
You got it!!

File: src/providers/ExchangeRateAPI.test.ts (hunk 1 above)
Comment (aaronmgdr) [reply src/providers/ExchangeRateAPI.test.ts:14]: the network response is as its a mock

Comment (Henrynunez112) [reply src/providers/ExchangeRateAPI.test.ts:14]: ahhh makes sense

**Pull request merged**